doc/source/conf.py

```diff
@@ -97,7 +97,7 @@
 html_show_sourcelink = True
 html_theme = "ansys_sphinx_theme"
 html_context = {
-    "github_user": "pyansys",
+    "github_user": "ansys",
```

Comment:
For this part I'm not sure where it is used, but it looks like the same change is needed.